packages/shared/src/lib/user.ts

```diff
@@ -80,6 +80,13 @@ export async function logout(): Promise<void> {
   });
 }
 
+export async function deleteAccount(): Promise<void> {
+  await fetch(`${apiUrl}/v1/users/delete`, {
+    method: 'POST',
+    credentials: 'include',
+  });
```

Comment:
What do you think about this suggestion?

```suggestion
  await fetch(`${apiUrl}/v1/users/me`, {
    method: 'DELETE',
    credentials: 'include',
  });
```

Reply:
Makes total sense, just noted we didn't use it anywhere assumed it might be a header issue :)